src/Testing/HomepageItem.test.js

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom';
```

Comment:
- [ ] As required for the project and for best practices, kindly make sure all your tests are passing as some of them are not passing like in the image below.
![Screenshot from 2023-06-06 21-57-09](https://github.com/Ashnaali3255/Stock-Metrics-WebApp/assets/83042742/84a64858-2517-46dd-8fde-fde9ff73bc16)

Reply:
@tresorsawasawa Thank you for your review but on my PC it works perfectly. If you don't mind Can you explain to me which test isn't working?

Reply:
Solved!!

![image](https://github.com/Ashnaali3255/Stock-Metrics-WebApp/assets/116655378/c9ab1f28-b109-48df-abab-e5eeec5de647)